ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+include::../shared-doc/attributes.adoc[]
+
+= ejb-txn-remote-call: Demonstrates remote EJB calls and transaction propagation
+:author: Ondra Chaloupka
+:level: Intermediate
+:technologies: EJB, JTA, Clustering
+
+[abstract]
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productName}.
+
+:standalone-server-type: ha
+:archiveType: war
+:uses-h2:
+:requires-multiple-servers:
+:jbds-not-supported:
+
+== What is it?
+
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productNameFull}.
+
+== Description
+
+The EJB remote calls contains a JTA transaction which is propagated over the EJB call.
+Further the quickstart demonstrates the transaction recovery which is run for both servers
+when a failure occurs.
+
+This quickstart contains two maven projects. As the goal is to make a remote call from one server
+to the another.
+The first maven projects is meant to be deployed on one server, representing the caller side,
```

Comment:
How about:

The first maven project represents the caller side, and is intended to be deployed on a single server. The second project represents the called side. This project is intended to be deployed to one or more additional servers.  The two projects should not be deployed to the same server.

Reply:
I used your sentence just adjuted it a little bit.